Source/Scene/ModelInstanceCollection.js

```diff
@@ -164,6 +165,8 @@ function ModelInstanceCollection(options) {
   this.specularEnvironmentMaps = options.specularEnvironmentMaps;
   this.backFaceCulling = defaultValue(options.backFaceCulling, true);
   this._backFaceCulling = this.backFaceCulling;
+  this.showOutline = defaultValue(options.showOutline, true);
+  this._showOutline = this.showOutline;
```

Comment:
`showOutline` needs to be passed through when the model is constructed. See [`modelOptions`](https://github.com/CesiumGS/cesium/blob/4fb9fbb097663898733ef5850f4df7d0174f3af0/Source/Scene/ModelInstanceCollection.js#L673-L697) - it should be added there